Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM lambci/lambda:build-nodejs8.10
+
+COPY . /var/task
```

Comment:
I did it this way in combination with `.dockerignore` instead of a `-v` mount just to avoid re-using node_modules installs cross-platform and potentially testing on dirty copies.